charts/redpanda/Chart.yaml

```
@@ -37,6 +37,10 @@ dependencies:
     condition: console.enabled
     version: ">=0.5 <1.0"
     repository: https://charts.redpanda.com
+  - name: connectors
+    condition: connectors.enabled
+    version: ">=0.1.2 <1.0"
+    repository: "file://../connectors"
```

`file` will only work if the user is running helm from within the checked-out git repository.

```suggestion
    repository: "https://charts.redpanda.com"
```

---

Yes, it will be addressed after https://github.com/redpanda-data/helm-charts/pull/649

---

Done